README.adoc

```diff
@@ -23,25 +23,26 @@
 :page-seo-description: A getting started tutorial and an example on how to enable distributed tracing in Java microservices to easily trace request flows that span multiple resources by using MicroProfile Telemetry and Jaeger distributed tracing system.​
 :source-highlighter: prettify
 :guide-author: Open Liberty
-= Enabling distributed tracing in microservices using OpenTelemetry observability framework 
+
+= Enabling distributed tracing in microservices by using the OpenTelemetry observability framework 
 
 [.hidden]
-NOTE: This repository contains the guide documentation source. To view the guide in published form, view it on the https://openliberty.io/guides/{projectid}.html[Open Liberty website].
+NOTE: This repository contains the guide documentation source. To view the guide in published form, view it at the https://openliberty.io/guides/{projectid}.html[Open Liberty website].
 
-Distributed tracing helps teams to keep track of requests between microservices. MicroProfile Telemetry adopts OpenTelemetry tracing to allow developers to observe requests across their distributed systems.
+Distributed tracing helps teams keep track of requests between microservices. MicroProfile Telemetry adopts OpenTelemetry tracing, allowing developers to observe requests across their distributed systems.
```

Comment:
since the audience for the guide is developers, use "you" to address reader directly
```suggestion
Distributed tracing helps teams keep track of requests between microservices. MicroProfile Telemetry adopts OpenTelemetry tracing, so you can observe requests across your distributed systems.
```